### PR TITLE
add opott.dino.icu

### PR DESCRIPTION
**Pull request by opott**
# Adding `opott.dino.icu`

## Description

I will be using this domain to host my personal portfolio site and have a custom email domain.

### CODE REVIEW

File: hackclub.com.yaml [hunk 1]
```diff
@@ -2822,4 +2822,19 @@ internal:
 - ttl: 600
   type: A
   value: 104.131.182.116
+
```
Comment (Muirrum) [inline hackclub.com.yaml:2825]: Hey! we don't allow personal sites on hackclub.com, please move this to dino.icu

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 1 above)
Comment (opott) [reply hackclub.com.yaml]: My bad! Changed this now!

File: dino.icu.yaml [hunk 2]
```diff
@@ -459,3 +459,8 @@ terror:
   ttl: 600 
   type: CNAME 
   value: 4H4CayJ.github.io.
+
+opott:
+- ttl: 600
+  type: CNAME
+  value: opott.pages.dev
```
Comment (Muirrum) [inline dino.icu.yaml:466]: In DNS, CNAME record values need a trailing dot, like this
```suggestion
  value: opott.pages.dev.
```

File: hackclub.com.yaml [hunk 3]
```diff
@@ -2822,4 +2822,3 @@ internal:
 - ttl: 600
   type: A
   value: 104.131.182.116
-
```
Comment (Muirrum) [inline hackclub.com.yaml:2825]: Could you revert your changes to this file entirely?

**Review by Muirrum — CHANGES_REQUESTED** [review]

File: hackclub.com.yaml (hunk 3 above)
Comment (opott) [reply hackclub.com.yaml:2825]: how do i do this?

**Comment by alx-alexpark** [comment]
Also, would suggest putting the record in alphabetical order in relation with the other records, even though not strictly required.

**Comment by opott** [comment]
hi! i will just close this pr and submit a new one.

**Pull request closed without merging**